plugins/orchestrator-backend/src/service/SonataFlowService.test.ts

```diff
@@ -75,47 +75,40 @@ describe('SonataFlowService', () => {
       global.fetch = jest.fn().mockResolvedValue(mockResponse as any);
 
       // When
-      const result = await sonataFlowService.fetchWorkflowInfoOnService({
-        definitionId,
-        serviceUrl,
-      });
-
-      // Then
-      expect(fetch).toHaveBeenCalledWith(urlToFetch);
-      expect(result).toBeUndefined();
-      expect(loggerMock.error).toHaveBeenCalledTimes(1);
-      expect(loggerMock.error).toHaveBeenCalledWith(
-        `Error when fetching workflow info: Error: ${await sonataFlowService.createPrefixFetchErrorMessage(urlToFetch, mockResponse as Response)}`,
-      );
-      expect(loggerMock.info).not.toHaveBeenCalled();
-      expect(loggerMock.debug).not.toHaveBeenCalled();
-      expect(loggerMock.warn).not.toHaveBeenCalled();
-      expect(loggerMock.child).not.toHaveBeenCalled();
+      let result;
+      try {
+        await sonataFlowService.fetchWorkflowInfoOnService({
+          definitionId,
+          serviceUrl,
+        });
+      } catch (error) {
+        result = error;
+      }
+
+      expect(result).toBeDefined();
```

Comment:
why do not use `rejects.toThrow(<expected error message>);`?

Reply:
it works so what's wrong with the current code?